wpilibcExamples/src/main/cpp/examples/RamseteCommand/include/RobotContainer.h

```diff
@@ -38,13 +35,11 @@ class RobotContainer {
   // The robot's subsystems
   DriveSubsystem m_drive;
 
+  // RobotContainer-owned commands
   frc2::InstantCommand m_driveHalfSpeed{[this] { m_drive.SetMaxOutput(0.5); },
                                         {}};
   frc2::InstantCommand m_driveFullSpeed{[this] { m_drive.SetMaxOutput(1); },
                                         {}};
 
-  // The chooser for the autonomous routines
-  frc::SendableChooser<frc2::Command*> m_chooser;
```

Comment:
Was this just never used?

Reply:
I think that a long time ago all examples had the chooser, and then it was mass-removed at some point, and this is a leftover.